### PR TITLE
Add Github Workflows CI

### PR DESCRIPTION
**Pull request by jackkoenig**
Just Wake 0.19.0 typechecking for now

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+    - name: Wit Init
+      uses: sifive/wit/actions/init@v0.13.2
+      with:
+        additional_packages: git@github.com:sifive/environment-blockci-sifive.git::0.7.0
+
+    - name: Wake Typecheck
+      uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
+      with:
+        command: -x Unit
```
Comment (terpstra) [inline .github/workflows/test.yml]: Actually, you don't need any of this fancy stuff. The default rule for actions/wake is '-x Unit' already. You don't need the additional_package if you are not going to run anything.

So:
```
steps:
- uses: sifive/wit/actions/init@v0.13.2
- uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
```
is all you actually need here.

**Review by terpstra — APPROVED** [review]
What you have is not wrong, and it might serve as a good foundation for more complicated rules if you forsee them being added in the future. However, for what you are doing, it's 8 lines more code than you needed.

File: .github/workflows/test.yml (hunk 1 above)
Comment (jackkoenig) [reply .github/workflows/test.yml]: Makes sense, still might be worth having names, what about:
```
steps:
- name: Wit Init
  uses: sifive/wit/actions/init@v0.13.2
- name: Wake Typecheck
  uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
```

Comment (terpstra) [reply .github/workflows/test.yml]: Entirely up to you. What you had is fine. What you propose now is fine.

Comment (jackkoenig) [inline .github/workflows/test.yml]:
```suggestion
    steps:
    - name: Wit Init
      uses: sifive/wit/actions/init@v0.13.2
    - name: Wake Typecheck
      uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
```

**Pull request merged**